diff --git a/src/components/SbcFooter.vue b/src/components/SbcFooter.vue
index 6fa4d2304bce52930cb6ed1007c108ba2dee30b2..15cacbeec27ff058eabaa3feb006416829ebc1d1 100644
--- a/src/components/SbcFooter.vue	
+++ b/src/components/SbcFooter.vue
@@ -4,7 +4,6 @@
       <nav>
         <ul>
           <li><a href="/">Home</a></li>
-          <li><a href="https://www.release-notes.bcregistry.gov.bc.ca" target="_blank">Release Notes</a></li>
           <li><a href="https://www2.gov.bc.ca/gov/content/home/disclaimer" target="_blank">Disclaimer</a></li>
           <li><a href="https://www2.gov.bc.ca/gov/content/home/privacy" target="_blank">Privacy</a></li>
           <li><a href="https://www2.gov.bc.ca/gov/content/home/accessibility" target="_blank">Accessibility</a></li>
diff --git a/src/util/interceptors.ts b/src/util/interceptors.ts
index 4db4ab9c6bc08453cd7448fb0027f8e3e5963ffc..8c4b038ac42c0d3d16015c4e0258eae182b89bf0 100644
--- a/src/util/interceptors.ts	
+++ b/src/util/interceptors.ts
@@ -4,6 +4,10 @@ import { SessionStorageKeys } from './constants'
 
 export function addAxiosInterceptors (axiosInstance: AxiosInstance): AxiosInstance {
   axiosInstance.interceptors.request.use(config => {
+    const authGatewayApiKey = import.meta.env.VUE_APP_AUTH_API_KEY
+    if (authGatewayApiKey && config.url.includes(ConfigHelper.getAuthAPIUrl())) {
+      config.headers['x-apikey'] = authGatewayApiKey
+    }
     const token = ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken)
     if (token) {
       config.headers.Authorization = `Bearer ${token}`
