diff --git a/src/util/interceptors.ts b/src/util/interceptors.ts
index 4db4ab9c6bc08453cd7448fb0027f8e3e5963ffc..8c4b038ac42c0d3d16015c4e0258eae182b89bf0 100644
--- a/src/util/interceptors.ts
+++ b/src/util/interceptors.ts
@@ -4,6 +4,10 @@ import { SessionStorageKeys } from './constants'
 
 export function addAxiosInterceptors (axiosInstance: AxiosInstance): AxiosInstance {
   axiosInstance.interceptors.request.use(config => {
+    const authGatewayApiKey = import.meta.env.VUE_APP_AUTH_API_KEY
+    if (authGatewayApiKey && config.url.includes(ConfigHelper.getAuthAPIUrl())) {
+      config.headers['x-apikey'] = authGatewayApiKey
+    }
     const token = ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken)
     if (token) {
       config.headers.Authorization = `Bearer ${token}`
